### PR TITLE
RDM-4955-1: Media Viewer - Functional tests

### PR DESCRIPTION
**Pull request by MKaramath**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-4955
https://tools.hmcts.net/jira/browse/RDM-6127

### Change description ###
Media Viewer - Functional tests 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: test/stepDefinitions/caseDetailsSteps.js [hunk 1]
```diff
@@ -123,4 +123,24 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
...
+    await caseDetailsPage.clickDocumentLink();
+  });
+
+  When(/^the media viewer is opened in a new tab$/, async function () {
+    let newTabUrl = await caseDetailsPage.getMediaViewerURL();
+    expect(newTabUrl.includes('/media-viewer')).to.true;
+  });
+
+  Then(/^the pdf document is visible in the new tab$/, async function () {
+    await caseDetailsPage.pdfContentVisible();
```
Comment (rafalkalita) [inline test/stepDefinitions/caseDetailsSteps.js]: Should this be
let visible = await caseDetailsPage.pdfContentVisible();
expect(visible).to.be.true;
?

File: test/stepDefinitions/caseDetailsSteps.js [hunk 2]
```diff
@@ -123,4 +123,24 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
...
+  });
+
+  Then(/^the image document is visible in the new tab$/, async function () {
+    await caseDetailsPage.imageContentVisible();
```
Comment (rafalkalita) [inline test/stepDefinitions/caseDetailsSteps.js]: Same as above.

File: test/stepDefinitions/caseDetailsSteps.js (hunk 1 above)
Comment (MSancaktutar) [reply test/stepDefinitions/caseDetailsSteps.js]: Of course. Thank you.

File: test/stepDefinitions/caseDetailsSteps.js (hunk 2 above)
Comment (MSancaktutar) [reply test/stepDefinitions/caseDetailsSteps.js]: Fixing now.

**Pull request merged**